docs/source/deployment/netlify.md

```diff
@@ -82,7 +82,8 @@ const resolvers = {
 
 const server = new ApolloServer({
   typeDefs,
-  resolvers
+  resolvers,
+  playground: true
```

Comment:
Fwiw, Playground is disabled automatically in Apollo Server when `process.env.NODE_ENV` is set to `production` unless `playground: true` is set explicitly as you've done here.  I assume `NODE_ENV=production` is set for a deployed Netlify Function, which explains why you couldn't get it to display without this change.

Forgive my unfamiliarity with the exact Netfliy workflow but is there not a Netlify Functions local development experience that has `NODE_ENV=development` (or `NODE_ENV` unset entirely?)

Reply:
Oh, I didn't know that. When I was walking through this, I was specifically trying to get the Playground to work on my local machine (not in production). 

Here's the blurb from the docs:
<img width="816" alt="image" src="https://user-images.githubusercontent.com/3806031/60897282-a440b980-a21c-11e9-9871-bc186f5c6a87.png">

So I don't know if adding `playground: true` is the best solution. We could either:

1) Tell the user to add `NODE_ENV=development` in the `package.json` for the `npm run start:lambda` command
2) Tell the user to add `playground: true` if they can't see the Playground while developing locally.

Reply:
I think I like the first option and would support that change!  Though if that fixes it, I am certainly curious what is setting `NODE_ENV` to `production` by default!